### PR TITLE
Microverse JavaScript Capstone

### PR DESCRIPTION
**Pull request by IsmaelMastronardi**
This webpage is an API-based web app created using the TVmaze API and the Microverse Involvement API. It displays movies, including a feature enabling the user to like a series and leave a comment about it.

- Movie cards created dynamically with JavaScript using information fetched from an API  ✅ 
- Each movie has a like button that you can click to add more likes to that movie  ✅ 
- A popup menu for every movie, that displays information about it ✅ 
- A comment section on the popup menu, where you can see all the comments and post a new comment ✅ 
- A counter for the number of movies that the page has available ✅ 
- A counter for the number of comments posted on every movie✅ 

https://github.com/IsmaelMastronardi/Capstone2/assets/128931752/28e899d4-0b33-49e3-872e-e46d608efdd8

### CODE REVIEW

File: src/Modules/popupMenu.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  <div class="comments" id="commentsDiv">  
+    <p id="firstComment"></p>
+  </div>
+  <form>
+    <title>Add A Comment</title>
+    <input id="fName" class="fName" placeholder="Your name">
+    <textarea id="fComment" class="fComment" placeholder="Your insigths"></textarea>
+    <button id="submitComment" class="submitBtn" type="button">Comment</button>
+  </form>
+</div>
```
Comment (OLIPLICHE) [inline src/Modules/popupMenu.js:43]: - [ ] Hi, The project requires that you should adhere to the given in terms of data displayed on the Page.  When I fill out the comment form with my `name` and `comment`, the app displays only the comment and hides the name. I would advise you to work out and fix the mistake. 
- [ ] Can you try to display the `date` because it is also part of the `data being displayed on the original page`.
I posted the pictures below for you to understand what I am talking about.
- **I fill out the Form:**
Capture

- **I submit the form**(NAME disappear)
Capture1
- **Original design**. 
Capture

File: src/test/commentCounterTest.test.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const commentCounter = require('./commentCounterTest.js');
```
Comment (OLIPLICHE) [inline src/test/commentCounterTest.test.js]: - [ ]  Hi, the counter functions here must be those functions that make your App to run(to work). I found out that you have created separate and isolated functions you have used as a test, and this is not what was requested.
- THEREFORE THE REQUIREMENT:
All the counter functions in the App need to be covered with unit tests using Jest. Make sure that you have tests for all edge cases.
* You should think at least about the scenario in which there are 0 items on the page (counter function should return 0)

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The App contains a `Home page header and navigation` 👍 
-  It contains a `Home page footer` 👍 
- Use ES6 and Webpack 👍 
- Use async and await with the fetch method 👍 

## Required Changes ♻️
- NB: It is a bad practice to work in the branch while your PR is still in review. I would advise you not to be doing so.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @IsmaelMastronardi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ All linters checks are ok
✅ The correct workflow is used. 
✅ Very descriptive Pull Request(PR)
✅ Your work is well documented.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
